src/pudl/transform/ferc1.py

```diff
@@ -1969,11 +1970,7 @@ def transform_main(self, df: pd.DataFrame) -> pd.DataFrame:
             .pipe(self.nullify_outliers)
             .pipe(self.replace_with_na)
             .pipe(self.drop_invalid_rows)
-            .pipe(
-                pudl.metadata.classes.Package.from_resource_ids()
-                .get_resource(self.table_id.value)
-                .encode
-            )
+            .pipe(PUDL_PACKAGE.encode)
```

Comment:
So encoding is applied to all FERC 1 tables already.

Reply:
this is an instance where i personally think it is fine to use the resource specific encoder bc we definitely know the resource name. but i don't feel very strongly about that so this is fine